_posts/0000-01-02-manojkumar44.md

```diff
@@ -0,0 +1,6 @@
+---
+layout:slide
+title:"Welcome to our second slide!"
+---
+Your text
```

Comment:
This is my latest website I've developed:
(https://www.purlconstructionltd.co.uk)